src/backend/distributed/utils/maintenanced.c

```diff
@@ -112,7 +112,7 @@ static volatile sig_atomic_t got_SIGHUP = false;
 static volatile sig_atomic_t got_SIGTERM = false;
 
 /* set to true when becoming a maintenance daemon */
-static bool IsMaintenanceDaemon = false;
+bool IsMaintenanceDaemon = false;
```

Comment:
I think this can stay static. there seems to be no file where this gets accessed directly, given the getter is added and exported.

Reply:
You are right, earlier, I did an export and removed static. Later, I added macro to get it, forgot to put the static back.